kratos/KratosMultiphysics.json

```diff
@@ -1,7 +1,7 @@
 {
 	"wheel_name": "KratosMultiphysics",
-	"included_modules": ["response_functions"],
-	"included_binaries": ["Kratos.*", "KratosCore.*", "KratosCore.*", "zlib.*", "libz.*"],
+    "included_modules":  ["response_functions"],
+	"included_binaries": ["Kratos.*", "KratosCore.*", "KratosCore.*", "KratosMPICore.*", "KratosMPI.*", "zlib.*", "libz.*"],
```

Comment:
this also works for a serial compilation?

Reply:
Yep, this is a filter list. It looks for the files in the folder matching any of those. If you don't compile it won't be copied.